.rubocop.yml

```diff
@@ -69,3 +69,7 @@ Rails/SkipsModelValidations:
   Exclude:
     - '**/app/read_models/**/*'
     - '**/lib/warden/**/*'
+
```

Comment:
Rubocop doesn't care which one you use, only that you are consistent. Here I've changed to block as it is more versatile when the object needs reloading or methods chaining.